pages/reference/api-support-matrix.en-US.mdx

```diff
@@ -90,15 +90,16 @@ import { Callout } from 'nextra-theme-docs';
 | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | On Demand upload format   | MP4                                                                                                                                                                                                |
 | Live latency              | 10-18 seconds with good RTMP ingest health, 2s segment size, and standard 3 seg buffering in player                                                                                                |
+| Low latency               | 3-5 seconds for transcoded rendition and sub-seconds for the source for browser-based applications                                                                                                 |
 | Live output format        | HLS playback URL (m3u8)                                                                                                                                                                            |
 | H264 Transcoding Profiles | Baseline, Main, High                                                                                                                                                                               |
 | Video players             | Supported tested: livepeer.js, video.js, hls.js, safari/ios native player, Theoplayer; Supported untested: JWPlayer, android native, Shaka, Exo, Bitmovin, Brightcove                              |
-| Video Delivery Location   | Supported tested: EU, USSupported untested: Asia, South America, Australia, Africa                                                                                                                 |
-| Live stream recording     | Only available for paying users. HLS playback URL (m3u8) available 5 minutes after session ends. Livepeer.com stores recording and serves playback via CDN. There is no limit on recording length. |
+| Video Delivery Location   | Supported tested: EU, US Supported untested: Asia, South America, Australia, Africa                                                                                                                 |
+| Live stream recording     | Only available for paying users. HLS playback URL (m3u8) is available 5 minutes after the session ends. Livepeer.studio stores recording and serves playback via CDN. The limit to the recording length is up to 12 hours max. |
```

Comment:
`livepeer.studio` -> `Livepeer Studio` - should always use the proper name in writing

> The limit to the recording length is up to 12 hours max. |

This is not technically true. Recommend rephrasing: 
```
When recording is enabled, `mp4` files are made available by default for recordings under 12 hours and are available on request for recordings over 12 hours. `m3u8` files are available for all recordings.
```